server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -1487,6 +1489,7 @@ abstract class Host(user: String,
 
     Try(CassandraLock().waitForModule(hosts(0), casSize))
     Try(ElasticsearchLock().waitForModule(hosts(0), esSize))
+    Thread.sleep(60000)
```

Comment:
as discussed with @e-orz we would like to retry when elastic index creation is failing and not to increase deployment time. please remove the sleep and add retry in template creation code.
BTW does only es is failing in docker or also cas?